.github/workflows/trymerge.yml

```diff
@@ -38,8 +38,13 @@ jobs:
           ON_GREEN: ${{ github.event.client_payload.on_green}}
           LAND_CHECKS: ${{ github.event.client_payload.land_checks }}
           COMMENT_ID: ${{ github.event.client_payload.comment_id }}
+          REBASE: ${{ github.event.client_payload.rebase }}
         run: |
           set -ex
+          if [ -n "${REBASE}" ]; then
+            python3 .github/scripts/tryrebase.py "${PR_NUM}"
```

Comment:
From https://github.com/pytorch/test-infra/pull/712: does it make sense to have REBASE contain the branch name, so we can run `python3 .github/scripts/tryrebase.py "${PR_NUM}" --branch "${REBASE}"`, with a default to master?

Reply:
done, it uses master as the default if -r is used, and which ever branch is specified if -r < branch > is used